src/Util/UrlUtil/UrlUtil.spec.js

```diff
@@ -151,6 +211,28 @@ describe('UrlUtil', () => {
         expect(got).toBe(requestString);
       });
     });
+
+    describe('#isValid', () => {
+      it('is defined', () => {
+        expect(UrlUtil.isValid).toBeDefined();
+      });
+      it('validates the given URL', () => {
+        const urls = {
+          'http://ows.terrestris.de': true,
+          'https://ows.terrestris.de': true,
+          'ftp://ows.terrestris.de': true,
+          'http://ows.terrestris.de:8080': true,
+          'http://ows.terrestris.de?param=key': true,
+          'http://ows.terrestris.de?param1=key1&param2=key2': true,
+          'http://10.133.7.9': true,
```

Comment:
how about testing

```
'http://foo@example.com/foo?humpty=dumpty',
'http://foo:s3cretP4ssw0rd@example.com/foo?humpty=dumpty'
```

do we allow these? I guess the rest of special cases like umlauts and encoded params are handled in the validator library, right?

Reply:
Yes, both urls are allowed, test cases added in 5dc63fe.

Reply:
:+1: